cli/parser/parser.go

```diff
@@ -732,6 +752,18 @@ func expandConfigs(workspaceDir string, args []string) ([]string, error) {
 	}
 	args = concat(args[:commandIndex+1], defaultArgs, args[commandIndex+1:])
 
+	enable, enableIndex, enableLength := arg.FindLast(args, "enable_platform_specific_config")
+	noEnable, noEnableIndex, noEnableLength := arg.FindLast(args, "noenable_platform_specific_config")
+	if enableIndex > noEnableIndex {
+		if enable == "true" || enable == "yes" || enable == "1" || enable == "" {
+			args = concat(args[:enableIndex], []string{"--config", getBazelOS()}, args[enableIndex+enableLength:])
+		}
+	} else if noEnableIndex > enableIndex {
+		if noEnable == "false" || noEnable == "no" || noEnable == "0" {
+			args = concat(args[:noEnableIndex], []string{"--config", getBazelOS()}, args[noEnableIndex+noEnableLength:])
+		}
```

Comment:
I could be understanding this wrongly, but validating `noEnable` value might not be needed.

```
> bazel build --noenable_platform_specific_config=no //...
ERROR: --noenable_platform_specific_config=no :: Unexpected value after boolean option: --noenable_platform_specific_config=no
```

Reply:
Oh, excellent. I didn't realize that that was how `no` flags worked. Thanks, fixed!